development/android-build-and-install-instructions.md

```diff
@@ -62,3 +62,31 @@ Debug version
 
 The debug version which you build yourself can be installed with the Android
 debugging bridge (adb).
+
+Android build server
+====================
+
+To build the release version on a build server, first follow the [building
+steps](#Building) above. `tools/android-prepare-api.sh` can easily take 15
+minutes, so make sure the it doesn't have to be called each build.
+
+Building release version
+------------------------
+
+To build a release version you'll need to supply the following parameters to
+`tools/android-build.sh`:
+
+1. `-r`: for release.
+
+2. `--ks`: path to the keystore containing the key which the apk will be signed
```

Comment:
how do we deal with this key in our workflow? where is it stored? Is it signed by someone?